src/widget/wcuemenupopup.cpp

```diff
@@ -27,6 +27,7 @@ WCueMenuPopup::WCueMenuPopup(QWidget* parent)
     m_pEditLabel = new QLineEdit(this);
     m_pEditLabel->setToolTip(tr("Edit cue label"));
     m_pEditLabel->setObjectName("CueLabelEdit");
+    m_pEditLabel->setPlaceholderText(tr("Cue Name..."));
```

Comment:
How about just "Label..."? "Cue" is a bit superfluous since it's the cue menu.

Reply:
done